infra/src/main/scala/scaladex/infra/github/GithubModel.scala

```diff
@@ -1,8 +1,13 @@
 package scaladex.infra.github
 
+import java.time.Instant
+
+import scala.util.Try
+
 import cats.implicits.toTraverseOps
 import io.circe._
 import io.circe.generic.semiauto._
+import org.joda.time.format.ISODateTimeFormat
```

Comment:
We should stop using `joda.time`. Here you should be able to use `DateTimeFormatter.ISO_DATE_TIME` form `java.time`.

```suggestion
import java.time.format.DateTimeFormatter
```